src/main/riddl/Venues/Locations/locationTypeDefinitions.riddl

```diff
@@ -0,0 +1,24 @@
+type Status is any of {Draft, Open, Closed} briefly "Internal status of a Location."
+
+type Info is {
+	id: LocationId,
+	name: String,
+  description: String,
+  directions: String briefly "How do you find this location?",
+  latLong: LatLong,
+  url: URL?,
+  capacity: Integer?,
+  accessible: Boolean briefly "Is this location handicapped accessible?",
```

Comment:
Is accessible an amenity? Or is it its own attribute? Your comment in the next line indicates that it is amenity.